src/component/environments/EnvironmentTable/EnvironmentIconCell/EnvironmentIconCell.tsx

```diff
@@ -0,0 +1,47 @@
+import { useContext, VFC } from 'react';
+import { styled } from '@mui/material';
+import { ConditionallyRender } from 'component/common/ConditionallyRender/ConditionallyRender';
+import { useSearchHighlightContext } from 'component/common/Table/SearchHighlightContext/SearchHighlightContext';
+import { Box, IconButton } from '@mui/material';
+import { CloudCircle, DragIndicator } from '@mui/icons-material';
+import { UPDATE_ENVIRONMENT } from 'component/providers/AccessProvider/permissions';
+import AccessContext from 'contexts/AccessContext';
+
+const DragIcon = styled(IconButton)(
+    ({ theme }) => `
+    padding: ${theme.spacing(0, 1, 0, 0)};
+    cursor: inherit;
+    color: ${theme.palette.grey[500]};
+    transition: color 0.2s ease-in-out;
+
+    [data-dragging=true] & {
+        color: ${theme.palette.grey[700]};
```

Comment:
Seems like a lot of logic for a slight color change, maybe we should just remove `isDragging` instead.